### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AndrasNemeth94**
@zsdominik  I have  done the code formatting with Prettier, alongside with basic structure and Header component, pls take a look at it.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}*/
```
Comment (zsdominik) [inline src/App.css]: Please remove this if you don't use them or if you use them then remove the comment.

https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

File: src/view/Footer.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.container-fluid{
+
```
Comment (zsdominik) [inline src/view/Footer.css]: remove the exra line pls

File: src/view/Footer.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                                </div>{/*----col copyright 2 end----- */}
+                            
+                        </div>{/*---row 5 end----- */}
+
+                {/*----container---*/}
+                </div>
+        );
+    }
+};
+export default Footer;
```
Comment (zsdominik) [inline src/view/Footer.js]: I think we can remove the unnecessary extra lines and please format the code correctly. Use Prettify or Eslint and set up the "Format on save" option in VSCode

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

File: src/view/Header.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.container-fluid{
+
```
Comment (zsdominik) [inline src/view/Header.css]: Extra line

File: src/view/Header.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                        </div>{/*---collapsed menu container end--- */}
+
+                        </div>{/*---menuCol end----- */}
+                </div>{/*---menuRow end---- */}
+                </nav>
+            </div>
+        );
+    }
+};
+export default Header;
```
Comment (zsdominik) [inline src/view/Header.js]: Same here with the formatter

File: src/view/Main.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+                    </div>{/*-----maindiv 2 end ----- */}
+
+                </div>{/*-----row end ----- */}
+
+            {/*-----Container end----- */}
+            </div>
+        );
+    }
+};
+export default Main;
```
Comment (zsdominik) [inline src/view/Main.js]: Same here with the formatter

**Review by zsdominik — CHANGES_REQUESTED** [review]
Please format the code correctly. Otherwise there is not much logic here yet. I think this will be good after you fix the format.

**Review by AndrasNemeth94 — COMMENTED** [review]
Thank you for the advice, I'll work on it

File: src/img/LoMLM.svg [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+     bordercolor="#666666"
+     borderopacity="1.0"
+     inkscape:pageopacity="0.0"
+     inkscape:pageshadow="2"
+     inkscape:zoom="0.7"
+<<<<<<< HEAD
+     inkscape:cx="134.11204"
+=======
+     inkscape:cx="372.68347"
+>>>>>>> 922b61a62dd7ad20bedb5e65b71b6c6c383bc722
```
Comment (zsdominik) [inline src/img/LoMLM.svg:51]: You did not resolve merge conflicts here and above inside this file.  Right now the code containing syntax errors and I'm pretty sure you are not able to display this svg correctly.

File: src/App.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+	render() {
+		return (
+			<div className="App">
+				<Header />
+				<Main />
+				<Footer />
+			</div>
+		);
+	}
+}
```
Comment (zsdominik) [inline src/App.js:19]: Sorry for the nitpicking about this but I think this tabulation is too much. In my opinion 2 or 4 spaces are enough.

check this example:
https://github.com/gothinkster/react-redux-realworld-example-app/blob/master/src/components/Editor.js#L95

File: src/view/Footer.css [hunk 9]
```diff
@@ -0,0 +1,9 @@
+.container-fluid {
+	background-color: black;
+}
+.copyright {
+	text-align: center;
+}
+p{
```
Comment (zsdominik) [inline src/view/Footer.css:7]: a space is missing after the p

File: src/view/Footer.js [hunk 10]
```diff
@@ -0,0 +1,116 @@
+
+import React, { Component } from "react";
+import "./Footer.css";
+
+class Footer extends Component {
+	render() {
+		return (
```
Comment (zsdominik) [inline src/view/Footer.js:7]: check the comment above about the indentation

File: src/view/Header.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+								<span className="linkText">CONTACT</span>
+							</ReactBootStrap.Nav.Link>
+						</ReactBootStrap.Nav>
+					</ReactBootStrap.Navbar.Collapse>
+				</div>
+				{/*---menuBar col end---- */}
+				{/*---collapse area end--- */}
+
+				{/*---navbar end--- */}
+			</ReactBootStrap.Navbar>
```
Comment (zsdominik) [inline src/view/Header.js:54]: check the comment above about the indentation

File: src/view/Main.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+						{/*-----maindiv 2 ----- */}
+					</div>
+					{/*-----maindiv 2 end ----- */}
+				</div>
+				{/*-----row end ----- */}
+
+				{/*-----Container end----- */}
+			</div>
+		);
+	}
```
Comment (zsdominik) [inline src/view/Main.js:31]: check the comment above about the indentation

**Review by zsdominik — CHANGES_REQUESTED** [review]

**Pull request closed without merging**